requirements-dev.txt

```diff
@@ -1,2 +1,3 @@
 nose2
 flake8
+dlint
```

Comment:
I can't remember. Does dlint get automatically bundled into flake8 or is there another step needed to link the two together.
ps. Thanks for the PR! We miss you!

Reply:
Dlint depends on flake8 - so installing Dlint will automatically install flake8. But yup, once Dlint is installed then flake8 will automatically detect its presence and run the additional linter rules. This is accomplished via an entrypoint in the Dlint Python package: https://github.com/duo-labs/dlint/blob/master/setup.py#L53

Hope all is well at Duo! :D

Reply:
Nice! I think this looks good then. I'm gonna go ahead and merge it in :)
Thanks again for making the PR and improving our linting even now ;)